include/OCPNPlatform.h

```diff
@@ -128,7 +128,25 @@ class OCPNPlatform
     wxString &GetSharedDataDir();
     wxString &GetPrivateDataDir();
     wxString GetWritableDocumentsDir();
+
+    /** The original in-tree plugin directory, sometimes not user-writable.*/
     wxString &GetPluginDir();
+
+    /**
+     * Base directory for user writable windows plugins, reflects
+     * winPluginDir option, defaults to %LOCALAPPDATA%/opencpn.
+     **/
+    wxString &GetWinPluginBaseDir();
+
+    /**
+     * Return ';'-separated list of directories to load plugins from. The
+     * list always includes the main installation directory. Some platforms
+     * prepends this with user-writable path(s), each of which ending in
+     * "opencpn". All paths are guaranteed to exist.
+     *
+     * For Linux, return paths ending in .../share. 
```

Comment:
With OpenCPN in /usr/local the plugin path is `/usr/local/share/opencpn/plugins`, does this return `/usr/local/share`?

Reply:
Thanks for review! The more eyes, the better...

For linux, this is as specified by the freeddesktop specs i. e. the contents of XDG_DATA_DIRS if existing, falling back to` ~/.local/share:/usr/local/share;/usr/share.`

Reply:
Ok, thanks for clarifying!

Reply:
It's also obiut the single directory used to install plugins in, and the list of directories which we load plugins from.  Wll update header docs on this, it's not even mentioned.